composer.json

```diff
@@ -21,7 +21,7 @@
         "amphp/parallel": "^2.1",
         "doctrine/collections": "^1.6.8 || ^2.0",
         "gitonomy/gitlib": "^1.3",
-        "laravel/serializable-closure": "^1.1",
+        "laravel/serializable-closure": "^1.1 || ^2.0",
```

Comment:
Is it worth supporting both? Supporting 2.x seems like a better approach to me - but I'm not sure how heavily people depend on this dependency in Laravel. Any clue?

Reply:
No idea :D 

I guess the biggest (or only) difference between 1 and 2 is they dropped PHP 7.x support, which GrumPHP support anyway so probably fine to go just with 2